src/components/withLocalize.js

```diff
@@ -115,6 +128,18 @@ class LocaleContextProvider extends React.Component {
         return LocalePhoneNumber.fromLocalPhone(this.props.preferredLocale, number);
     }
 
+    isValidLocaleDigit(digit) {
+        return LocaleDigitUtils.isValidLocaleDigit(this.props.preferredLocale, digit);
+    }
+
+    toLocaleDigit(digit) {
+        return LocaleDigitUtils.toLocaleDigit(this.props.preferredLocale, digit);
+    }
+
+    fromLocaleDigit(localeDigit) {
+        return LocaleDigitUtils.fromLocaleDigit(this.props.preferredLocale, localeDigit);
+    }
+
```

Comment:
I can't think of any other place in the app that will use these functions so I don't think it is a good idea to pollute the WithLocalize context at this time. Maybe directly use them in the IOUAmountPage for now and pass the locale from `this.props.preferredLocale`. We can refactor these in future if needed.

Reply:
Currently `toLocaleDigit` is also used by `BigNumberPad`.

Reply:
Also, I cleaned up the `isValidLocaleDigit` function because it's not used. So only `toLocaleDigit` and `fromLocaleDigit` is added in that context.